CHANGELOG.md

```diff
@@ -7,6 +7,34 @@
 - Added new resource
   - xSQLServerAlwaysOnAvailabilityGroupDatabaseMembership
   - Added localization support for all strings.
+  - Moved the resource to the DSCClassResources directory.
+  - Moved the Import-SQLPSModule command from the constructor to the Set method.
+- Changes to xSQLServerRSConfig
```

Comment:
Could you rebase this one against the new base branch test-problem-class-based-resource-issue-#673